release/release.go

```diff
@@ -309,7 +501,7 @@ func addToTarball(tw *tar.Writer, dst, src string) {
 	header := &tar.Header{
 		Name: dst,
 		Size: stat.Size(),
-		Mode: 0755,
+		Mode: int64(stat.Mode()),
```

Comment:
why this change? in the PR where this was added, @xdg had requested that we use a hard-coded mode instead of relying on the source file mode

Reply:
Not all of the files are executable. It seems pretty weird to me to make like README.md executable.

Reply:
But, up to you and @xdg